ara/ui/templates/partials/task_index_row.html

```diff
@@ -0,0 +1,61 @@
+{% load datetime_formatting %}
+{% load truncatepath %}
+{% load static_url %}
+<tr>
+  <td class="text-center">
+    {% include "partials/task_status_icon.html" with status=task.status %}
+  </td>
+  <td class="text-center">
+    {% url 'ui:playbook' task.playbook.id as playbook_url %}
+    <a role="button" class="btn btn-outline-primary btn-link btn-sm" href="{% static_url playbook_url %}?task={{ task.id }}#results">
+      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-box-arrow-up-right" viewBox="0 0 16 16">
+        <path fill-rule="evenodd" d="M8.636 3.5a.5.5 0 0 0-.5-.5H1.5A1.5 1.5 0 0 0 0 4.5v10A1.5 1.5 0 0 0 1.5 16h10a1.5 1.5 0 0 0 1.5-1.5V7.864a.5.5 0 0 0-1 0V14.5a.5.5 0 0 1-.5.5h-10a.5.5 0 0 1-.5-.5v-10a.5.5 0 0 1 .5-.5h6.636a.5.5 0 0 0 .5-.5z"></path>
+        <path fill-rule="evenodd" d="M16 .5a.5.5 0 0 0-.5-.5h-5a.5.5 0 0 0 0 1h3.793L6.146 9.146a.5.5 0 1 0 .708.708L15 1.707V5.5a.5.5 0 0 0 1 0v-5z"></path>
+      </svg>
+    </a>
+  </td>
+  <td class="text-nowrap">{{ task.started | format_date }}</td>
+  <td class="text-nowrap">{{ task.duration | format_duration }}</td>
+  <td class="text-center">{{ task.items.results }}</td>
+  <td class="text-center">
+    <span class="badge badge-pill badge-secondary" class="btn btn-secondary" data-toggle="tooltip"  data-placement="right" data-html="true" title="{% if task.tags %}{% include 'partials/tags_list.html' with tags=task.tags %}{% endif %}">
+      {{ task.tags | length }}
+    </span>
+  </td>
+  <td>
+    <a {% if not static_generation %}href="{% url 'ui:task_index' %}?action={{ task.action }}" title="Search tasks for {{ task.action }}"{% endif %}>
+      {{ task.action }}
+    </a>
+  </td>
+  <td>
+    <a {% if not static_generation %}href="{% url 'ui:task_index' %}?name={{ task.name }}" title="Search tasks for {{ task.name }}"{% endif %}>
+      {{ task.name }}
+    </a>
+  </td>
+  <td class="text-nowrap" title="{{ task.path }}">
+    {% if not static_generation %}
+      <a href="{% url 'ui:task_index' %}?path={{ task.path }}">{{ task.path | truncatepath:25 }}</a>
+      <strong>:</strong>
+      <a href="{% url 'ui:task_index' %}?path={{ task.path }}&lineno={{ task.lineno }}">{{ task.lineno }}</a>
+    {% else %}
+        {{ playbook.path | truncatepath:25 }}:{{ task.lineno }}
+    {% endif %}
+  </td>
+  {% if task.playbook.name is not None %}
+  <td title="Search results for {{ task.playbook.name }}">
+    {% if not static_generation %}
+      <a href="{% url 'ui:task_index' %}?playbook_name={{ task.playbook.name }}">{{ task.playbook.name }}</a>
+    {% else %}
+      {{ task.playbook.name }}
+    {% endif %}
+  </td>
+  {% else %}
+  <td title="Search results for {{ task.playbook.path }}">
+    {% if not static_generation %}
+      <a href="{% url 'ui:task_index' %}?playbook_path={{ task.playbook.path }}">{{ task.playbook.path | truncatepath:25 }}</a>
```

Comment:
Thinking out loud: I wonder whether this is the right link for this.

tl;dr: change from ``?playbook_path=playbook.yml`` to ``?playbook_path=playbook.yml&name=install%20apache`` ?

Right now clicking on a playbook's name (or path) would return every tasks for every playbook that matches the playbook name (or path).

Instead, it could provide a search for that task's name in addition to the playbook name (or path) such that it only provided this tasks' results across every playbook that match the playbook name (or path). There could be false positives due to the icontains nature of the search but it should otherwise be fairly precise.

Reply:
I thought about it and let's go with the current implementation.
We can always make the search more precise later if we change our mind.